### PR TITLE
Feature/192 user home responsive

### PR DESCRIPTION
**Pull request by sadiabbasi**

### CODE REVIEW

File: src/demo/components/ImageSlider.css [hunk 1]
```diff
@@ -37,11 +37,11 @@
 }
 .right-side-image {
   text-align: left;
-  max-width: 400px !important;
+  max-width: 400px ;
 
 }
 .left-side-image img, .right-side-image img {
-  height: 70% !important;
```
Comment (theblockstalk) [inline src/demo/components/ImageSlider.css:44]: good you remove this! bad practice to use `!important`

File: src/demo/components/ImageSlider.css [hunk 2]
```diff
@@ -138,3 +138,88 @@ line-height: 35px !important;
...
+    min-width: 90%;
+    margin-left: -2.5rem;
+  }
+  .left-side-image,
+  .right-side-image {
+    text-align: center;
+    max-width: 100%;
+  }
+
+  .left-side-image img,
```
Comment (theblockstalk) [inline src/demo/components/ImageSlider.css:182]: we should use hyphon-separated-text or camelCase for classes, but not both. Please choose one and stick to it.

File: src/demo/components/ImageSlider.css [hunk 3]
```diff
@@ -138,3 +138,88 @@ line-height: 35px !important;
...
+  .right-side-image img {
+    height: auto;
+    width: 100%;
+  }
+
+  .centerImageText {
```
Comment (theblockstalk) [inline src/demo/components/ImageSlider.css:188]: this is camelCase

File: src/demo/components/CodeSnippetPreview.css [hunk 4]
```diff
@@ -73,4 +73,9 @@
     justify-content: center; 
     gap: 56px; 
   }
-  
\ No newline at end of file
+  
+  @media only screen and (min-width: 375px) and (max-width: 767px) {
+    .display-none{
```
Comment (theblockstalk) [inline src/demo/components/CodeSnippetPreview.css:78]: maybe better to call this `mobile-display-none` to make it more clear

Comment (sadiabbasi) [reply src/demo/components/CodeSnippetPreview.css:78]: we can use this for mobile plus web like when we add logout icon on top view of mobile may be we can hide it on web view.

**Review by theblockstalk — APPROVED** [review]

**Pull request merged**